daprdocs/content/en/reference/components-reference/supported-cryptography/local-storage.md

```diff
@@ -0,0 +1,38 @@
+---
+type: docs
+title: "Local storage"
+linkTitle: "Local storage"
+description: Detailed information on the local storage cryptography component
+---
+
+## Component format
+
+Todo: update component format to correct format for cryptography
+
+A Dapr `crypto.yaml` component file has the following structure:
+
```

Comment:
Here perhaps we can add a note, shared by all built-in crypto components, something like:

```md
This component uses the built-in cryptographic engine in Dapr to perform operations. Although keys are never exposed to your application, Dapr has access to the raw key material.
```